cfgov/unprocessed/js/routes/on-demand/simple-chart/simple-chart.js

```diff
@@ -218,14 +218,16 @@ function addProjectedMonths(chartObject, numMonths) {
   /* Add a zone to each series with a dotted line starting
      at the projected data starting point */
   chartObject.series = chartObject.series.map((singluarSeries) => {
+    let projectedStyle = { dashStyle: 'dot' };
+    if (chartObject.chart && chartObject.chart.type === 'column') {
```

Comment:
You might be able to shorten this using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) `if (chartObject.chart?.type === 'column') {`